src/include/sof/audio/buffer.h

```diff
@@ -191,6 +191,34 @@ struct buffer_cb_free {
 struct comp_buffer *buffer_alloc(uint32_t size, uint32_t caps, uint32_t flags, uint32_t align,
 				 bool is_shared);
 struct comp_buffer *buffer_new(const struct sof_ipc_buffer *desc, bool is_shared);
+#if CONFIG_ZEPHYR_DP_SCHEDULER
+/*
+ * create a shadow dp_queue buffer before buffer (when as_source == true_
+ * or behind a buffer
```

Comment:
'_' should be a closing parenthesis? Also, what exactly do you mean by "before" or "behind" a buffer?

Reply:
comment added